ansible_ai_connect/ai/api/telemetry/schema2.py

```diff
@@ -82,3 +83,21 @@ class AnalyticsProductFeedback:
     timestamp: str = field(
         default=Factory(lambda self: timezone.now().isoformat(), takes_self=True)
     )
+
+
+@frozen
+class OneClickTrialPlan:
+    created_at: str = field(validator=validators.instance_of(str), converter=str, default="")
+    expired_at: str = field(validator=validators.instance_of(str), converter=str, default="")
+    is_active: bool = False
+    name: str = field(validator=validators.instance_of(str), converter=str, default="")
+    id: int = 0
```

Comment:
The default to `0` is wrong. `field(validator=validators.instance_of(int))` should be enough

Reply:
done